webknossos/webknossos/client/_upload_dataset.py

```diff
@@ -25,55 +24,67 @@ def _cached_get_upload_datastore(context: _WebknossosContext) -> str:
     raise ValueError("No datastore found where datasets can be uploaded.")
 
 
+def _walk(
+    path: Path, base_path: Optional[Path] = None
+) -> Iterator[Tuple[Path, Path, int]]:
+    if base_path is None:
+        base_path = path
+    if path.is_dir():
+        for p in Path(path).iterdir():
```

Comment:
One more minor thing I just found:
```suggestion
        for p in path.iterdir():
```